server/src/internalClusterTest/java/org/elasticsearch/indices/recovery/IndexRecoveryIT.java

```diff
@@ -782,21 +782,24 @@ public Settings onNodeStopped(String nodeName) {
      * Tests shard recovery throttling on the target node. Node statistics should show throttling time on the target node, while no
      * throttling should be shown on the source node because the target will accept data more slowly than the source's throttling threshold.
      */
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/103204")
     public void testTargetThrottling() throws Exception {
         logger.info("--> starting node A with default settings");
-        final String nodeA = internalCluster().startNode();
+        final String nodeA = internalCluster().startNode(
+            Settings.builder()
+                .put(RecoverySettings.INDICES_RECOVERY_MAX_BYTES_PER_SEC_SETTING.getKey(), randomIntBetween(10, 200), ByteSizeUnit.MB)
```

Comment:
I'm avoiding the "[crazy slow](https://github.com/elastic/elasticsearch/blob/main/test/framework/src/main/java/org/elasticsearch/test/InternalTestCluster.java#L565)" values here since w/o unthrottling it would take a longer time and we time out on green.

Reply:
I think even 10MiB/s might be slow enough to risk being throttled sometimes, if the shard is more than 50kiB in size (see [this comment](https://github.com/elastic/elasticsearch/issues/103204#issuecomment-1849513308) for the calculation). IMO let's set it "crazy fast" to just avoid all this.

Reply:
The limit we set on nodeB is based on that (half of what the rate limiter considers throttle-able). How can it still happen? (We could go higher, but to me it seems this is just to avoid super slow recovery hitting the timeout, not to accidentally cause throttling on source).